packages/upgrade/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should match the eslintrc in the other packages as much as possible

Reply:
https://github.com/nomad-xyz/monorepo/pull/349/commits/1f6674e7d2098d6253fbbe7420c492d690bced9b